From fe2de3b565c685e5ec87e8d1def67da22c0504bb Mon Sep 17 00:00:00 2001
From: Carlos Garnacho <carlosg@gnome.org>
Date: Sun, 06 Feb 2011 13:33:44 +0000
Subject: Handle better panel applets.

A gradient was clobbering the background-color set in
the css rules for the panel applets. Fixes bug #641425,
reported by Bastien Nocera.
---
diff --git a/themes/Adwaita/gtk-3.0/gtk.css b/themes/Adwaita/gtk-3.0/gtk.css
index f70368b..157d8e0 100644
--- a/themes/Adwaita/gtk-3.0/gtk.css
+++ b/themes/Adwaita/gtk-3.0/gtk.css
@@ -684,7 +684,9 @@ TerminalScreen {
 /* For old gnome-panel */
 PanelWidget,
 PanelApplet,
+PanelMenuBar.menubar,
 PanelToplevel {
     background-color: #1e1a17;
+    background-image: none;
     color: #fff;
 }
--
cgit v0.8.3.1
